aten/src/ATen/native/sparse/cuda/SparseBlasImpl.cpp

```diff
@@ -795,6 +795,56 @@ void addmm_out_sparse_csr(
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~linalg_solve~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+template <typename scalar_t>
+void _apply_sparse_csr_lu_solve(
+  const Tensor& input,
+  const Tensor& other,
+  Tensor& result,
+  int& _singularity) {
+  auto values = input.values();
+  const scalar_t *values_data_ptr = values.data_ptr<scalar_t>();
+  auto crow_indices = input.crow_indices().to(kInt);
+  const int *crow_indices_data_ptr = crow_indices.data_ptr<int>();
+  auto col_indices = input.col_indices().to(kInt);
+  const int *col_indices_data_ptr = col_indices.data_ptr<int>();
+  cusolverSpHandle_t _handle;
+  cusolverSpCreate(&_handle);
+  auto descrA = at::cuda::sparse::CuSparseMatDescriptor();
+
+  const scalar_t *b = other.data_ptr<scalar_t>();
+  int n = crow_indices.numel() - 1; 
+  int nnzA = input._nnz();
+  float tol = 0;
+  int reorder = 1;
+  scalar_t *x = result.data_ptr<scalar_t>();
+  int singularity = _singularity;
+  at::cuda::sparse::linear_solve(_handle, n, nnzA, descrA, values_data_ptr,
+    crow_indices_data_ptr, col_indices_data_ptr, tol, reorder, x, &singularity);
```

Comment:
Error tells us that the problem is in `descrA`:
```
no matching function for call to 'linear_solve(..., at::cuda::sparse::CuSparseMatDescriptor&, ...)
note: candidate: 'template<class scalar_t> void at::cuda::sparse::linear_solve(..., cusparseMatDescr_t, ...)
```
Try passing `descrA.descriptor()` instead of `descrA`.